tests/ipex/test_modeling.py

```diff
@@ -255,6 +259,37 @@ def test_pipeline(self, model_arch):
         self.assertEqual(pipe.device, model.device)
         self.assertTrue(all("This is a sample" in item["generated_text"] for item in outputs))
 
+    @parameterized.expand(
+        grid_parameters(
+            {
+                "model_arch": IPEX_PATCHED_SUPPORTED_ARCHITECTURES,
+                "use_cache": [True, False],
+                "num_beams": [1, 4],
+                "batch_size": [1, 4],
```

Comment:
no need to export the model multiple times for different `num_beams` and `batch_size`, I think we can iterate after the export directly